doc/quickstart.rst

```diff
@@ -16,7 +16,7 @@ To install the dependencies::
 Alternatively to using pip, on Debian stretch or later you can install
 dependencies from the repository::
 
-    apt-get install python3-dateutil python3-django python3-celery \
+    apt-get install python3-dateutil python3-django python3-celery libyaml-dev \
```

Comment:
note the paragraph above says "alternatively to using pip", so at this point it's already too late. we probably need a new paragraph before that, or an extra line before `pip install` with the required -dev  packages. One probably also wants libpq-dev for postgres, etc.